app/questionnaire/placeholder_parser.py

```diff
@@ -107,7 +107,10 @@ def __call__(
         return self._placeholder_map
 
     def _get_value_source_resolver(
-        self, routing_path_block_ids: OrderedSet[str] | None = None
+        self,
+        *,
+        routing_path_block_ids: OrderedSet[str] | None = None,
+        assess_routing_path: bool = False,
```

Comment:
Existing issue, but why is the default False?. Should it not be `None`?. False has a different meaning to None when used in VSR.

Reply:
Updated to default to be None 👍 and updated type hinting

Reply:
This default was correct actually. By default, placeholder don't need to assess the path, so False is valid.